applications/nrf_desktop/doc/passkey.rst

```diff
@@ -1,4 +1,4 @@
-.. _passkey:
+.. _passkey_module:
```

Comment:
Other documentation pages do not have "_module" suffix in the link. Can i revert this change?